KitePrintSDK/src/main/java/ly/kite/address/Country.java

```diff
@@ -1,343 +1,501 @@
+    TAJIKISTAN                        ( "Tajikistan", "TJ", "TJK", "TJS", false ),
+    TANZANIA                          ( "Tanzania, United Republic of", "TZ", "TZA", "TZS", false ),
+    THAILAND                          ( "Thailand", "TH", "THA", "THB", false ),
+    TIMOR_LESTE                       ( "Timor-Leste", "TL", "TLS", "USD", false ),
+    TOGO                              ( "Togo", "TG", "TGO", "XOF", false ),
+    TOKELAU                           ( "Tokelau", "TK", "TKL", "NZD", false ),
+    TONGA                             ( "Tonga", "TO", "TON", "TOP", false ),
+    TRINIDAD_AND_TOBAGO               ( "Trinidad and Tobago", "TT", "TTO", "TTD", false ),
+    TUNISIA                           ( "Tunisia", "TN", "TUN", "TND", false ),
+    TURKEY                            ( "Turkey", "TR", "TUR", "TRY", false ),
+    TURKMENISTAN                      ( "Turkmenistan", "TM", "TKM", "TMT", false ),
+    TURKS_AND_CAICOS_ISLANDS          ( "Turks and Caicos Islands", "TC", "TCA", "USD", false ),
+    TUVALU                            ( "Tuvalu", "TV", "TUV", "AUD", false ),
+    UGANDA                            ( "Uganda", "UG", "UGA", "UGX", false ),
+    UKRAINE                           ( "Ukraine", "UA", "UKR", "UAH", true ),
+    UAE                               ( "United Arab Emirates", "AE", "ARE", "AED", false ),
+    UK                                ( "United Kingdom", "GB", "GBR", "GBP", true ),
+    USA                               ( "United States", "US", "USA", "USD", false ),
+    USA_MINOR_OUTLYING_ISLANDS        ( "United States Minor Outlying Islands", "UM", "UMI", "USD", false ),
+    URUGUAY                           ( "Uruguay", "UY", "URY", "UYU", false ),
+    UZBEKISTAN                        ( "Uzbekistan", "UZ", "UZB", "UZS", false ),
+    VANUATU                           ( "Vanuatu", "VU", "VUT", "VUV", false ),
+    VENEZUELA                         ( "Venezuela, Bolivarian Republic of", "VE", "VEN", "VEF", false ),
+    VIET_NAM                          ( "Viet Nam", "VN", "VNM", "VND", false ),
+    BRITISH_VIRGIN_ISLANDS            ( "Virgin Islands, British", "VG", "VGB", "USD", false ),
+    US_VIRGIN_ISLANDS                 ( "Virgin Islands, U.S.", "VI", "VIR", "USD", false ),
+    WALLIS_AND_FUTUNA                 ( "Wallis and Futuna", "WF", "WLF", "XPF", false ),
+    WESTERN_SAHARA                    ( "Western Sahara", "EH", "ESH", "MAD", false ),
+    YEMEN                             ( "Yemen", "YE", "YEM", "YER", false ),
+    ZAMBIA                            ( "Zambia", "ZM", "ZMB", "ZMW", false ),
+    ZIMBABWE                          ( "Zimbabwe", "ZW", "ZWE", "ZWL", false);
+
+
+  ////////// Static Constant(s) //////////
+
+  @SuppressWarnings( "unused" )
+  private static final String  LOG_TAG = "TemplateClass";
+
+
+  ////////// Static Variable(s) //////////
+
+  private static Hashtable<String,Country>  sCodeCountryTable;
```

Comment:
Hashtable is a synchronised (i.e. thread safe) version of HashMap: I 
either had a good reason for using it at the time (that I obviously 
forgot to document), or I just didn't realise I had typed it. Was it a 
Hashtable before?

On 20/07/15 11:08, Deon Botha wrote:
>
> In KitePrintSDK/src/main/java/ly/kite/address/Country.java 
> <https://github.com/OceanLabs/Android-Print-SDK/pull/9#discussion_r34981677>:
>
> > +    WALLIS_AND_FUTUNA                 ( "Wallis and Futuna", "WF", "WLF", "XPF", false ),
> > +    WESTERN_SAHARA                    ( "Western Sahara", "EH", "ESH", "MAD", false ),
> > +    YEMEN                             ( "Yemen", "YE", "YEM", "YER", false ),
> > +    ZAMBIA                            ( "Zambia", "ZM", "ZMB", "ZMW", false ),
> > +    ZIMBABWE                          ( "Zimbabwe", "ZW", "ZWE", "ZWL", false);
> > +
> > +
> > +  ////////// Static Constant(s) //////////
> > +
> > +  @SuppressWarnings( "unused" )
> > +  private static final String  LOG_TAG = "TemplateClass";
> > +
> > +
> > +  ////////// Static Variable(s) //////////
> > +
> > +  private static Hashtable<String,Country>  sCodeCountryTable;
>
> Out of interest, why the choice to use |Hashtable| vs |HashMap|?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/OceanLabs/Android-Print-SDK/pull/9/files#r34981677>.
>